### DISCUSSION TITLE
Does Seq apply any sampling to OpenTelemetry traces

### DISCUSSION THREAD

**Discussion by ChristopherL-STCU** [datalust/seq-tickets#2297] [Help] [answered]
When sending OpenTelemetry traces to Seq does Seq apply any sort of sampling, or does it ingest every trace?

**Comment by nblumhardt** [comment, marked as answer]
Hi @ChristopherL-STCU,

Seq doesn't apply any sampling server-side, you'll need to use a client-side sampler for this (let me know if you need a hand configuring the OTel client).

Hope this helps,
Nick
